content/authors/Rahmani/_index.md

```diff
@@ -0,0 +1,59 @@
+---
+# Display name
+title: Rahmani
+
+# Is this the primary user of the site?
+superuser: false
+
+# Role/position
+role: Postdoctoral Scholar
+
+# Organizations/Affiliations
+organizations:
+- name: USC QCB
+  url: "https://dornsife.usc.edu/qcb/"
+
+# Short bio (displayed in user profile at end of posts)
+bio: I am interested in computational approaches and biophysical modeling to study genome organization.
+
+# Social/Academic Networking
+# For available icons, see: https://sourcethemes.com/academic/docs/page-builder/#icons
+#   For an email link, use "fas" icon pack, "envelope" icon, and a link in the
+#   form "mailto:your-email@example.com" or "#contact" for contact widget.
+social:
+
+#- icon: envelope
+#  icon_pack: fas
+#  link: 'mailto:test@example.org'
+
+- icon: twitter
```

Comment:
you can delete the twitter icon if you don't want to link to it

Reply:
Done.